Code/autopkglib/MunkiImporter.py

```diff
@@ -84,6 +84,13 @@ class MunkiImporter(Processor):
                 "Extension for output pkginfo files. Default is 'plist'.",
             "required": False
         },
+        "metadata_additions": {
+            "description": (
+                "A dictionary that will be added directly into the _metadata "
+                "of the pkginfo"
```

Comment:
A small quibble, but perhaps it should be more explicit about overlapping keys. Something like:

A dictionary that will be merged with the pkginfo _metadata. Unique keys will be added to the _metadata; existing keys will be replaced.

You understand this, and I understand this, but I guarantee the question will be asked.